api/src/test/java/org/openmrs/validator/ObsValidatorTest.java

```diff
@@ -513,4 +516,36 @@ public void validate_shouldPassValidationIfValueTextIsLessThanTheMaximumLength()
 		Assert.assertFalse(errors.hasFieldErrors("obsDatetime"));
 		Assert.assertFalse(errors.hasFieldErrors("valueText"));
 	}
+
+	/**
+	 * @see ObsValidator#validate(Object, Errors)
+	 */
+	@Test
+	//@Verifies(value = "should add person with encounter patient for a new obs", method = "saveObs(Obs,String)")
+	public void saveObs_shouldUpdateObsPersonValueWithEncounterPatientForNewObs() {
```

Comment:
You need to rename the method to match the fact that you're checking logic in the ObsValidator